changelog.md

```diff
@@ -78,6 +78,8 @@
 
 - Removed the `nimIncrSeqV3` define.
 
+- Added the `nimLegacyTypeMismatch` switch to get legacy type mismatch error messages.
```

Comment:
We don't implement new switches via defines.

Reply:
I will add it to legacy feature then.